src/configs/battery.rs

```diff
@@ -24,17 +25,42 @@ impl<'a> Default for BatteryConfig<'a> {
             unknown_symbol: "",
             empty_symbol: "",
             format: "[$symbol$percentage]($style) ",
-            display: vec![BatteryDisplayConfig {
-                threshold: 10,
-                style: "red bold",
-            }],
+            display: vec![BatteryDisplayConfig::default()],
             disabled: false,
         }
     }
 }
 
-#[derive(Clone, ModuleConfig, Default, Serialize)]
+#[derive(Clone, Serialize)]
 pub struct BatteryDisplayConfig<'a> {
     pub threshold: i64,
     pub style: &'a str,
+    pub symbol: Option<&'a str>,
+}
+
+impl<'a> Default for BatteryDisplayConfig<'a> {
+    fn default() -> Self {
+        BatteryDisplayConfig {
+            threshold: 10,
+            style: "red bold",
+            symbol: None,
+        }
+    }
+}
```

Comment:
Can you add the new defaults to the documentation table, or keep this as `derive(Default)`, so the defaults are obvious?

Reply:
I added the default value to the documentation table.